oak-search-elastic/pom.xml

```diff
@@ -252,6 +258,14 @@
       <groupId>org.apache.jackrabbit</groupId>
       <artifactId>oak-jcr</artifactId>
       <version>${project.version}</version>
+      <classifier>tests</classifier>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.jackrabbit</groupId>
+      <artifactId>oak-search</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
```

Comment:
Same as lucene - is test-jar needed here ?

Reply:
same.